lib/cancel-stuck-txs.js

```diff
@@ -0,0 +1,171 @@
+import pRetry from 'p-retry'
+import assert from 'node:assert'
+import * as Sentry from '@sentry/node'
+import ms from 'ms'
+import timers from 'node:timers/promises'
+
+const ROUND_LENGTH_MS = ms('20 minutes')
+
+/**
+  * @param {string} f4addr
+  * @returns 1000 oldest messages
+  */
+async function getMessagesInMempool (f4addr) {
+  const res = await pRetry(
+    () => fetch(`https://filfox.info/api/v1/message/mempool/filtered-list?address=${f4addr}&pageSize=1000`),
+    {
+      async onFailedAttempt (error) {
+        console.warn(error)
+        console.warn('Filfox request failed. Retrying...')
+      }
+    }
+  )
+  if (!res.ok) {
+    throw new Error(`Filfox request failed with ${res.status}: ${(await res.text()).trimEnd()}`)
+  }
+  /** @type {{
+     messages: {
+      cid: string;
+      from: string;
+      to: string;
+      nonce: number;
+      value: string;
+      gasLimit: number;
+      gasFeeCap: string;
+      gasPremium: string;
+      method: string;
+      methodNumber: number;
+      evmMethod: string;
+      createTimestamp: number;
+    }[];
+  }}
+  */
+  const { messages } = /** @type {any} */(await res.json())
+  return messages
+}
+
+/**
+ * @returns {Promise<{
+  "cid": string;
```

Comment:
_Nitpick - feel free to ignore._

We usually don't put property names into (double quotes).

```suggestion
  cid: string;
```

The same comment applies to all properties below too.

Reply:
I have copy pasted this from your PR, will adjust